backend/timed/redmine/management/commands/redmine_report.py

```diff
@@ -53,6 +53,11 @@ def handle(self, *args, **options):
             Project.objects.filter(id__in=affected_projects)
             .order_by("name")
             .annotate(total_hours=Sum("tasks__reports__duration"))
+            .annotate(
+                billable_hours=Sum(
+                    "tasks__reports__duration", filter=Q(not_billable=False)
```

Comment:
@c0rydoras This `Q` wants to filter the Projects, but I need to filter the `tasks`. Idk the django database model too well. Is there a way to do this "quick & dirty"?
Or should I just make an additional query fort tasks lower down in the `for project...` and filter for project and billability?

I *think* once this works the MR is good to go, unless you're not happy with it only be tested in one of the two tests.